### ISSUE TITLE
Stream and File extensions in conflict if stream/protocols config includes "file"

### ISSUE THREAD

**Issue by jodal** [mopidy/mopidy#1248]
From https://discuss.mopidy.com/t/update-to-mopidy-1-1-mopidy-is-not-starting/835:

```
Aug 11 07:38:43 musik mopidy[24629]: File "/usr/lib/python2.7/dist-packages/mopidy/core/actor.py", line 166, in __init__
Aug 11 07:38:43 musik mopidy[24629]: ) % (scheme, name(b), name(backends_by_scheme[scheme]))
Aug 11 07:38:43 musik mopidy[24629]: AssertionError: Cannot add URI scheme file for FileBackend, it is already handled by StreamBackend
```

Lots of users have set the `stream/protocols` config themselves, so they still have `file` as a protocol that Mopidy-Stream should handle. As of Mopidy 1.1.0, we include a new Mopidy-File backend that claims the `file` protocol. This causes a conflict between the extensions, and Mopidy shuts down.

As a workaround for this quite common issue, we should filter out `file` from the values in `stream/protocols`.

**Comment by adamcik** [comment]
We might also want to make this conditional on `file` being enabled.

**Comment by jodal** [comment]
Fixed by #1254.
